diagnostic.md

```diff
@@ -184,5 +216,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+165px
```

Comment:
at 550px, the `@media(min-width= 400px)` would take over so `.stretch` is 25%. Thus 25% * 550px = 137.50px.